### PR TITLE
refactor(lib): refactor templates

### PR DESCRIPTION
**Pull request by natusaspire**

### CODE REVIEW

File: src/lib/application/files/ts/src/main.ts [hunk 1]
```diff
@@ -1,8 +1,11 @@
 import { NestFactory } from '@nestjs/core';
+import { INestApplication, INestExpressApplication } from '@nestjs/common';
 import { AppModule } from './app.module';
 
-async function bootstrap() {
-  const app = await NestFactory.create(AppModule);
+async function bootstrap(): Promise<void> {
+  const app: INestApplication & INestExpressApplication = await NestFactory.create(AppModule);
```
Comment (kamilmysliwiec) [inline src/lib/application/files/ts/src/main.ts]: could you get rid of these additional typings? we'll change it very soon in the core

File: src/lib/application/files/js/src/main.js [hunk 2]
```diff
@@ -4,6 +4,8 @@ import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+
```
Comment (kamilmysliwiec) [inline src/lib/application/files/js/src/main.js]: is this one extra line really required? I would suggest staying consistent with the docs

**Comment by kamilmysliwiec** [comment]
I left to small comments. The rest looks good :) Thanks

File: src/lib/application/files/ts/src/main.ts (hunk 1 above)
Comment (natusaspire) [reply src/lib/application/files/ts/src/main.ts]: Done.

File: src/lib/application/files/js/src/main.js (hunk 2 above)
Comment (natusaspire) [reply src/lib/application/files/js/src/main.js]: Updated.

**Comment by kamilmysliwiec** [comment]
LGTM. Thanks!

**Pull request merged**